Week3/homework/2-step3.js

```diff
@@ -1,8 +1,41 @@
 'use strict';
+let range = function (startIndex ,stopIndex) {
```

Comment:
Why did you use a function expression instead of a function statement here? It seems inconsistent with the other function definitions in this file.